hazelcast-spring/src/main/resources/hazelcast-spring-3.6.xsd

```diff
@@ -2415,4 +2434,17 @@
     <xs:simpleType name="wan-ack-type">
         <xs:union memberTypes="wan-ack-type-format-enum non-space-string"/>
     </xs:simpleType>
+
+    <xs:complexType name="hot-restart">
+        <xs:sequence>
+            <xs:element name="home-dir" type="xs:string" minOccurs="0" maxOccurs="1" default="hot-restart"/>
+        </xs:sequence>
+        <xs:attribute name="enabled" type="xs:string">
```

Comment:
@bilalyasar, @serkan-ozal: I see. `parameterized-boolean` may be a better choice. But I don't want to start a build just to change an XSD definition. 

Also, spring schema seems very inconsistent in this matter. Some elements are using `parameterized-X` types, some are just `xs:string`.